### ISSUE TITLE
Error while training Custom Named Entity Recognition with BERT

### ISSUE THREAD

**Issue by Onkar-2803** [NielsRogge/Transformers-Tutorials#74]
![temp](https://user-images.githubusercontent.com/60955023/152527682-ffd91071-ad43-4929-ad3b-dd0cd9c4e4bb.PNG)

**Comment by NielsRogge** [comment]
Hi,

Thanks for your interest in my notebook. I made that notebook more than a year ago, and I should actually update it, because currently, models in Transformers return a dictionary (in case of token classification, actually a `TokenClassifierOutput`) rather than a tuple.

Hence, the correct way to get the loss is as follows:

```
outputs = model(input_ids=input_ids, attention_mask=mask, labels=labels)
loss = outputs.loss
```

**Comment by Onkar-2803** [comment]
Hey!

Thanks a lot! It worked, I was also trying to find the updated functions and check what it returns.

For anyone who's trying to run this code in the latest version, besides the above-mentioned change also make the following edits-
In " class dataset(Dataset): " change the parameter passed to self.tokenizer from "is_pretokenized" to "is_split_into_words"

**Comment by NielsRogge** [comment]
Thanks for raising this issue, I've updated my fine-tuning BERT for NER notebook.
